lib/godmin/helpers/forms.rb

```diff
@@ -4,6 +4,11 @@ module Forms
       def form_for(record, options = {}, &block)
         super(record, { builder: FormBuilders::FormBuilder, inline_errors: false }.merge(options), &block)
       end
+
+      def select_tag(name, option_tags = nil, options = {})
```

Comment:
Unless I'm mistaken, `select_tag` is a view helper: 
- http://api.rubyonrails.org/classes/ActionView/Helpers/FormTagHelper.html#method-i-select_tag
- https://github.com/rails/rails/blob/dd7af2c413a06ea44e50abf0df205314ba1bfc98/actionview/lib/action_view/helpers/form_tag_helper.rb#L131

Therefore, it makes sense to override it to be visually coherent with the rest of the UI.

Note that this is only for cases where a `form_tag` is required, not a `form_for`.

This is also justified by the fact that there seems to be no way to pass a form builder object to the `form_tag` helper (unless I got that wrong).

http://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html

Another approach could be to apply the missing attribute via javascript, but then the `selectize` initialization logic would get more complicated.